server/controller/course.controller.js

```diff
@@ -376,15 +188,101 @@ courseController.delete(
  */
 courseController.delete(
   '/all',
-  passport.authenticate('jwt', { session: false }),
+  // passport.authenticate('jwt', { session: false }),
```

Comment:
Make sure this is uncommented as it is crucial the user is authenticated before deleting all data. Though eventually this would only be authorized for the admin to delete an entire collection, but for now just a part of testing.